src/main/java/mate/academy/spring/online/bookstore/exception/CustomGlobalExceptionHandler.java

```diff
@@ -42,4 +43,14 @@ public ResponseEntity<Object> handleEntityNotFoundException(
         body.put("error", ex.getMessage());
         return new ResponseEntity<>(body, HttpStatus.NOT_FOUND);
     }
+
+    @ExceptionHandler(RegistrationException.class)
+    public ResponseEntity<Object> handleRegistrationException(
+            RegistrationException ex,
+            WebRequest request) {
+        Map<String, Object> body = new LinkedHashMap<>();
+        body.put("status", HttpStatus.CONFLICT);
```

Comment:
sorry for confusing, I remove timestamp by mistake in previous comment
```suggestion
        body.put("timestamp", LocalDateTime.now());
```